backend/de.metas.deliveryplanning.base/src/main/java/de/metas/deliveryplanning/DeliveryPlanningRepository.java

```diff
@@ -85,6 +85,15 @@ protected I_M_Delivery_Planning getById(@NonNull final DeliveryPlanningId delive
 		return load(deliveryPlanningId, I_M_Delivery_Planning.class);
 	}
 
+	public List<I_M_Delivery_Planning> getByReleaseNo(@NonNull final String releaseNo)
+	{
+		return queryBL.createQueryBuilder(I_M_Delivery_Planning.class)
+				.addOnlyActiveRecordsFilter()
+				.addEqualsFilter(I_M_Delivery_Planning.COLUMNNAME_ReleaseNo, releaseNo)
```

Comment:
do we haven an index on releaseno? if not, please add one (no need to add it in the AD..just a simple index would be enough for me)

Reply:
yes, there is. See: https://github.com/metasfresh/metasfresh/blob/0d94a4c31b9201a80c477b75be652afd7a688ad0/backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5672360_sys_gh14215_M_Delivery_Planning_ReleaseNo_Index.sql